config/sidebar-learn.json

```diff
@@ -383,6 +383,11 @@
 			"title": "Experimental",
 			"slug": "experimental",
 			"routes": [
+					{
+							"source": "learn/experimental/overview.mdx",
+							"label": "Overview",
+							"slug": "overview"
+					},
```

Comment:
DANGER: we used to have a page with this exact name and slug. That means there is a chance that we created a redirect when we deleted it, which would cause problems for you now. I dunno if it's the case, but something to check out before merging.